apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.IntegrationTests/BigtableFixture.cs

```diff
@@ -24,12 +23,14 @@
 
 namespace Google.Cloud.Bigtable.V2.IntegrationTests
 {
+    // Note: this use CloudProjectFixtureBase as a base class, as an emulator can be used as an alternative.
```

Comment:
This sentence, I fear, may contain some grammatical errors.

Reply:
Oops. Thanks, will fix. Will also actually *run* the integration tests to check I haven't messed them up...